self-serve-storage/pure-r/daaas_storage.R

```diff
@@ -9,12 +9,10 @@ get_bash_variable <- function (location, var) {
         intern = TRUE
     )
 }
-### Just sets the environment variables.
-daaas_storage.__getClient__<- function(storage_type = c("standard", "premium")) {
-
-    type <- match.arg(storage_type)
+# Just sets the environment variables.
+daaas_storage.get_client<- function(instance) {
     env_vars <- c("MINIO_URL", "MINIO_ACCESS_KEY", "MINIO_SECRET_KEY")
-    location <- sprintf("/vault/secrets/minio-%s-tenant-1", type)
+    location <- sprintf("/vault/secrets/%s", instance)
```

Comment:
`sprintf` in R is a wrapper around `sprintf` in C which is  known to be dangerous.

https://stackoverflow.com/questions/3662899/understanding-the-dangers-of-sprintf

Maybe try a different approach for string substitution and printing.